### ISSUE TITLE
Farlight 84 (1928420)

### ISSUE THREAD

**Issue by catemiko** [ValveSoftware/Proton#6717]
# Compatibility Report
- Name of the game with compatibility issues: Farlight 84
- Steam AppID of the game: 1928420

## System Information
- GPU:  AMD Radeon Graphics (AMD Ryzen 5 5500U)
- Driver/LLVM version: Mesa 23.2.0
- Kernel version: 6.2.12-zen1-1-zen
- Link to full system information report as: [Gist](https://gist.github.com/xRoBoTx/aeb07ff835bc42533584b4f04d21566c)
- Proton version: 8.0-1

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-1928420.log](https://github.com/ValveSoftware/Proton/files/11323325/steam-1928420.log)

## Symptoms 

Game crashes after launch.
![image](https://user-images.githubusercontent.com/28825932/234302922-9592e91d-5667-4433-a5b2-378dbbc9c622.png)

## Reproduction

1. Download the game
2. Launch the game

**Comment by kisak-valve** [comment]
Hello @xRoBoTx, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

Looking at your Proton log, `info:    Driver : AMD open-source driver 2.0.249` looks like you're using AMDVLK, then later in the log we see `info:  Null hardware (RADV NAVI10):` which looks like DXVK and RADV failing to find your video card. Can you test the game without AMDVLK in the mix?

**Comment by catemiko** [comment]
> Hello @xRoBoTx, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.
> 
> Looking at your Proton log, `info: Driver : AMD open-source driver 2.0.249` looks like you're using AMDVLK, then later in the log we see `info: Null hardware (RADV NAVI10):` which looks like DXVK and RADV failing to find your video card. Can you test the game without AMDVLK in the mix?

Well, i there is no AMDVLK in my system. BUT, after you said it i tried to install it and figured out that actually helped me! Lol!
So its working with AMDVLK now!

**Comment by kisak-valve** [comment]
Your system information from 15 minutes ago disagrees:  https://gist.github.com/xRoBoTx/aeb07ff835bc42533584b4f04d21566c#file-steam-sysinfo-log-L290-L306 and https://gist.github.com/xRoBoTx/aeb07ff835bc42533584b4f04d21566c#file-steam-sysinfo-log-L431-L460 shows 32 bit amdvlk most likely from the `lib32-amdvlk` system package.

**Comment by catemiko** [comment]
I had `lib32-vulkan-radeon` package.

**Comment by ghost** [comment]
Looks like today's update broke the game on Linux (Game windows open with a black screen, then closes after a couple of seconds)

# Compatibility Report
- Name of the game with compatibility issues: Farlight 84
- Steam AppID of the game: 1928420

## System Information
- GPU: NVIDIA GeForce RTX 3060
- Video driver version: nvidia 535.86.05
- Kernel version: 5.19.0-50-generic
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/gabuch2/fc563715f85daf89e2761238a3e8ab84
- Proton version: Proton Experimental / Same behavior with Proton GE 

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.



## Symptoms 
The game after showing the EAC screen, shows a black screen then closes. This didn't happen before the latest Farlight update. 

## Reproduction
1. Open the Game



[steam-1928420.log](https://github.com/ValveSoftware/Proton/files/12180201/steam-1928420.log)

**Comment by lucasgabmoreno** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/6717#issuecomment-1653016372

Same problem here:
Nvidia 940m and same drivers, etc.

**Comment by Gustavo17pacheco** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/6717#issuecomment-1653016372

Same problem:
RX 570 on latest Fedora Mesa drivers.

**Comment by wendersonp** [comment]
![Uploading Screenshot_20230801-082157~3.png…]()
My guess is the whole issue is related to this

**Comment by wendersonp** [comment]
![Screenshot_20230801-082157~3](https://github.com/ValveSoftware/Proton/assets/8678263/12873a5a-7487-42e0-9113-eb3c4eef9eba)
My guess is, the whole issue is related to this.

**Comment by lucasgabmoreno** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/6717#issuecomment-1660122424

Thanks!
I've tried uninstall completely, remove Fairlight 84 Steam folder, reinstall and use PROTON_EAC="path to Proton Easy Anti Cheat" %command and still can't load the game, just first screen, then black screen and close.

**Comment by GloriousEggroll** [comment]
This may be some kind of GPU driver bug. I tested on 4 systems all running nobara. Steam Deck worked, laptop with 6800m worked, desktop with gnome and 7900xtx failed, media center with 7900 xtx and kde failed. The failure is the same noted in the other proton logs in this issue by others: 
```
`[0803/182652.376:ERROR:network_change_notifier_win.cc(156)] WSALookupServiceBegin failed with: 8
[0803/182652.776:ERROR:mf_helpers.cc(12)] Error in dxva_video_decode_accelerator_win.cc on line 355
10.0.19043.1466
pid 11591 != 11590, skipping destruction (fork without exec?)`
```

**Comment by ghost** [comment]
The update released a few days ago fixed this issue, [they also explicitly stated it was working with the Steam Deck](https://store.steampowered.com/news/app/1928420/view/3645152842266011485).

**Comment by GloriousEggroll** [comment]
> The update released a few days ago fixed this issue, [they also explicitly stated it was working with the Steam Deck](https://store.steampowered.com/news/app/1928420/view/3645152842266011485).

still crashing here on two different systems, both with 7900 xtx.

**Comment by massatt212** [comment]
even with the latest update its still wont go pass the Start Game screen.


[steam-1928420.log](https://github.com/ValveSoftware/Proton/files/12327455/steam-1928420.log)

Also my Nvidia PC crashes also.

**Comment by Etaash-mathamsetty** [comment]
fixed with this patch: 
[farlight84_patch.txt](https://github.com/ValveSoftware/Proton/files/12383524/farlight84_patch.txt)

it's very much a hack and some debugging has to be done to figure out why USERPROFILE is not defined
if anyone wants, I can upload a build of proton-tkg with this patch applied

proton build: https://github.com/Etaash-mathamsetty/wine-builds/releases/tag/farlight84

**Comment by lucasgabmoreno** [comment]
> fixed with this patch: [farlight84_patch.txt](https://github.com/ValveSoftware/Proton/files/12383524/farlight84_patch.txt)
> 
> it's very much a hack and some debugging has to be done to figure out why USERPROFILE is not defined if anyone wants, I can upload a build of proton-tkg with this patch applied
> 
> proton build: https://github.com/Etaash-mathamsetty/wine-builds/releases/tag/farlight84

Not working on last update "Farlight 84 V14.4.1.5 Hotfix Notes"

**Comment by massatt212** [comment]
It's down again, you tried with proton experimental?

**Comment by Etaash-mathamsetty** [comment]
looks like this is a game bug, I don't see anything crashing on wine side

**Comment by ghost** [comment]
I can confirm the game closes again with the latest patch.

**Comment by lucasgabmoreno** [comment]
Proton 8 and Experimental, same error:
```
[0822/200715.624:ERROR:network_change_notifier_win.cc(156)] WSALookupServiceBegin failed with: 8
[0822/200719.085:ERROR:browser_gpu_channel_host_factory.cc(103)] Failed to launch GPU process.
```

**Comment by Etaash-mathamsetty** [comment]
> Proton 8 and Experimental, same error:
> 
> ```
> [0822/200715.624:ERROR:network_change_notifier_win.cc(156)] WSALookupServiceBegin failed with: 8
> [0822/200719.085:ERROR:browser_gpu_channel_host_factory.cc(103)] Failed to launch GPU process.
> ```

huh I don't get that second error

**Comment by fde-capu** [comment]
Here is a contribution to help solving the problem. Last week, Farlight 84 was running fine on my system, until the last update, August 22, V14.4.1.5, when I now have: first screen > black screen (when the video is supposed to start) > crash. I believe the problem due to the hardware acceleration for the video player used on the loading intro, because of the dxva error I get running proton by cli:

`[0825/012303.681:ERROR:mf_helpers.cc(12)] Error in dxva_video_decode_accelerator_win.cc on line 355`

I am using Proton 8.0-3 as compatibility. No changes on my system since last week. It was working fine! Now it is not.

Specs:

```
Computer Information:
Manufacturer: ASUSTeK COMPUTER INC.
Model: Z170-A
Form Factor: Desktop
No Touch Input Detected
Processor Information:
CPU Vendor: GenuineIntel
CPU Brand: Intel(R) Core(TM) i7-6700 CPU @ 3.40GHz
CPU Family: 0x6
CPU Model: 0x5e
CPU Stepping: 0x3
CPU Type: 0x0
Speed: 4000 MHz
8 logical processors
4 physical processors
Hyper-threading: Supported
FCMOV: Supported
SSE2: Supported
SSE3: Supported
SSSE3: Supported
SSE4a: Unsupported
SSE41: Supported
SSE42: Supported
AES: Supported
AVX: Supported
AVX2: Supported
AVX512F: Unsupported
AVX512PF: Unsupported
AVX512ER: Unsupported
AVX512CD: Unsupported
AVX512VNNI: Unsupported
SHA: Unsupported
CMPXCHG16B: Supported
LAHF/SAHF: Supported
PrefetchW: Unsupported
Operating System Version:
Debian GNU/Linux 12 (bookworm) (64 bit)
Kernel Name: Linux
Kernel Version: 6.1.0-11-amd64
X Server Vendor: The X.Org Foundation
X Server Release: 12101007
X Window Manager: Mutter (Muffin)
Steam Runtime Version: steam-runtime_0.20230606.51628
Video Card:
Driver: NVIDIA Corporation NVIDIA GeForce GTX 970/PCIe/SSE2
Driver Version: 4.6.0 NVIDIA 525.125.06
OpenGL Version: 4.6
Desktop Color Depth: 24 bits per pixel
Monitor Refresh Rate: 59 Hz
VendorID: 0x10de
DeviceID: 0x13c2
Revision Not Detected
Number of Monitors: 1
Number of Logical Video Cards: 1
Primary Display Resolution: 3840 x 2160
Desktop Resolution: 3840 x 2160
Primary Display Size: 27.48" x 15.47" (31.54" diag), 69.8cm x 39.3cm (80.1cm diag)
Primary Bus: PCI Express 16x
Primary VRAM: 4096 MB
Supported MSAA Modes: 2x 4x 8x 16x
Sound card:
Audio device: Realtek ALC892
Memory:
RAM: 15931 Mb
VR Hardware:
VR Headset: None detected
Miscellaneous:
UI Language: English
LANG: en_US.UTF-8
Total Hard Disk Space Available: 261344 MB
Largest Free Hard Disk Block: 161232 MB
Storage:
Number of SSDs: 5
SSD sizes: 480G,480G,320G,250G,160G
Number of HDDs: 0
Number of removable drives: 0
```

BTW I am using this script I have GTP-generated: https://pastebin.com/zHL7hLZf

**Comment by lucasgabmoreno** [comment]
> > Proton 8 and Experimental, same error:
> > ```
> > [0822/200715.624:ERROR:network_change_notifier_win.cc(156)] WSALookupServiceBegin failed with: 8
> > [0822/200719.085:ERROR:browser_gpu_channel_host_factory.cc(103)] Failed to launch GPU process.
> > ```
> 
> huh I don't get that second error

Must be my Nvidia.

**Comment by massatt212** [comment]
Check and see if it's working on steamdeck, if not the game is broken on Linux again, and will have to wait till they fix it, so tell the dev in the steam discussion section for farlight

**Comment by jorgicio** [comment]
> 

I know people with AMD GPUs and also crashes after loading the EAC screen. So that happened to me.
The game broke again after the 22nd August update.

**Comment by simifor** [comment]
Crash after EAC screen is still happening, tried proton 8.0.3 and bleeding edge 
[steam-1928420.log](https://github.com/ValveSoftware/Proton/files/12705149/steam-1928420.log)
These lines seem relevant:
```
[0922/195642.317:ERROR:mf_helpers.cc(12)] Error in dxva_video_decode_accelerator_win.cc on line 355
err:   D3D11VideoDevice::GetVideoDecoderProfileCount: Stub
```

**Comment by ghost** [comment]
Still crashing on the newest Proton version (8.0-4)

[steam-1928420.log](https://github.com/ValveSoftware/Proton/files/12839413/steam-1928420.log)

**Comment by massatt212** [comment]
I think we all have up on this game.

**Comment by Etaash-mathamsetty** [comment]
yeah im still convinced this is a game bug, but I have to check again

**Comment by JupiterSky11** [comment]
Was working just yesterday, now it isn't working after the 15.1 update.  Successful booting seems to be influenced by the phase of the moon.

**Comment by rezzafr33** [comment]
There are at least 2 workarounds :
1. add `SteamDeck=1` envar
```
SteamDeck=1 %command%
```
2. disable `RichTapWinSDKDLL.dll`
```
WINEDLLOVERRIDES="RichTapWinSDKDLL.dll=d" %command%
```
Another way is by launching steam with `-steamdeck` argument which I think is the same as first method.

ps: only for desktop linux

**Comment by davispuh** [comment]
I was getting black screen but after disabling `RichTapWinSDKDLL.dll` it works fine :)

**Comment by Etaash-mathamsetty** [comment]
actual fix: https://gitlab.winehq.org/wine/wine/-/merge_requests/4428

**Comment by lucasgabmoreno** [comment]
Now it works, but random freeze all computer when playing.
I've nvidia GTX 1650, Corei-10 10°

**Comment by lucasgabmoreno** [comment]
> Now it works, but random freeze all computer when playing. I've nvidia GTX 1650, Corei-10 10°

Solved with Nvidia performance mode + PROTON_NO_FSYNC=1 PROTON_NO_ESYNC=1

**Comment by rezzafr33** [comment]
Doesn't seem to work anymore, there is another error:
```
13391.062:018c:0190:err:module:LdrInitializeThunk "llhpbase.dll" failed to initialize, aborting
13391.062:018c:0190:err:module:LdrInitializeThunk Initializing dlls for L"Z:\\mnt\\games\\SteamLibrary\\steamapps\\common\\Farlight 84\\WindowsClient\\Solarland\\Binaries\\Win64\\SolarlandClient-Win64-Shipping.exe" failed, status c0000005
```
[steam-1928420.log](https://github.com/ValveSoftware/Proton/files/13626321/steam-1928420.log)

**Comment by lucasgabmoreno** [comment]
> Doesn't seem to work anymore, there is another error:
> 
> ```
> 13391.062:018c:0190:err:module:LdrInitializeThunk "llhpbase.dll" failed to initialize, aborting
> 13391.062:018c:0190:err:module:LdrInitializeThunk Initializing dlls for L"Z:\\mnt\\games\\SteamLibrary\\steamapps\\common\\Farlight 84\\WindowsClient\\Solarland\\Binaries\\Win64\\SolarlandClient-Win64-Shipping.exe" failed, status c0000005
> ```
> 
> [steam-1928420.log](https://github.com/ValveSoftware/Proton/files/13626321/steam-1928420.log)

Use [Proton GE](https://github.com/GloriousEggroll/proton-ge-custom) (GloriousEggroll). Install it using [ProtonUp-Qt](https://flathub.org/apps/net.davidotek.pupgui2)

**Comment by wendersonp** [comment]
Game is broken on Valve's protons, but works using Proton GE. It should not be verified for the steam deck

**Comment by SteavenGamerYT** [comment]
game is now broken

**Comment by ziomek64** [comment]
Yep, game is not working. Support is telling on discord that it is, obvious bs

**Comment by xpander69** [comment]
WINE_DO_NOT_CREATE_DXGI_DEVICE_MANAGER=1 %command%

seems to make it work at least with GE-Proton9-7

**Comment by ziomek64** [comment]
> WINE_DO_NOT_CREATE_DXGI_DEVICE_MANAGER=1 %command%
> 
> seems to make it work at least with GE-Proton9-7

I'm getting abnormal environment detected when in match instantly. But I can get to menu at least.

EDIT: Nvm, proton-ge is working, idk why mine from cachyOS isn't.

**Comment by Etaash-mathamsetty** [comment]
> WINE_DO_NOT_CREATE_DXGI_DEVICE_MANAGER=1 %command%
> 
> seems to make it work at least with GE-Proton9-7

I thought no games need that anymore

**Comment by ziomek64** [comment]
> > WINE_DO_NOT_CREATE_DXGI_DEVICE_MANAGER=1 %command%
> > 
> > seems to make it work at least with GE-Proton9-7
> 
> I thought no games need that anymore 

What's it for usually?

**Comment by jorgicio** [comment]
> WINE_DO_NOT_CREATE_DXGI_DEVICE_MANAGER=1 %command%
> 
> seems to make it work at least with GE-Proton9-7

I confirm this setting works.

**Comment by simifor** [comment]
farlight should work with the current proton experimental bleeding edge, and without extra environment variables.

**Comment by ziomek64** [comment]
> farlight should work with the current proton experimental bleeding edge, and without extra environment variables.

- What changed?

**Comment by Etaash-mathamsetty** [comment]
> > farlight should work with the current proton experimental bleeding edge, and without extra environment variables.
> 
> - What changed?

Gofman added the workaround to the proton script

**Comment by ziomek64** [comment]
> > > farlight should work with the current proton experimental bleeding edge, and without extra environment variables.
> > 
> > - What changed?
> 
> Gofman added the workaround to the proton script 

I see, will this come to proton ge if eggroll updates it?

**Comment by SteavenGamerYT** [comment]
steam is no longer compiling shaders for this game

**Comment by simifor** [comment]
@SteavenGamerYT Do you mean the pop up that appears after clicking play but before the game launches? It's working fine over here. Do you have shader precaching disabled(settings > downloads)? Also note that you will only need to recompile shaders when either the game or your drivers get an update.

**Comment by SteavenGamerYT** [comment]
> @SteavenGamerYT Do you mean the pop up that appears after clicking play but before the game launches? It's working fine over here. Do you have shader precaching disabled(settings > downloads)? Also note that you will only need to recompile shaders when either the game or your drivers get an update.

It's enabled and the game Is sttuering like crazy and dropping fps I have dxvk hud enabled and it keeps saying compiling shaders while playing this drops fps

**Comment by SteavenGamerYT** [comment]
> @SteavenGamerYT Do you mean the pop up that appears after clicking play but before the game launches? It's working fine over here. Do you have shader precaching disabled(settings > downloads)? Also note that you will only need to recompile shaders when either the game or your drivers get an update.

It's enabled and the game Is sttuering like crazy and dropping fps I have dxvk hud enabled and it keeps saying compiling shaders while playing this drops fps

**Comment by jorgicio** [comment]
Be careful about switching Proton versions: it may trigger some anticheat system.

I did it so yesterday and got banned until March 1st for "using third party modified programs" (I was just playing on Linux).

Also I had to switch a proton version because I was using Experimental and crashed, so I moved to 9 stable.

**Comment by Dezzel** [comment]
> Be careful about switching Proton versions: it may trigger some anticheat system.
> 
> I did it so yesterday and got banned until March 1st for "using third party modified programs" (I was just playing on Linux).
> 
> Also I had to switch a proton version because I was using Experimental and crashed, so I moved to 9 stable.

Now it doesn't matter if you change the proton or not, you get banned about once a week for a few days.

**Comment by VelorumS** [comment]
In battle royale mode it crashes after 5 seconds repeatedly.

**Comment by LiamDawe** [comment]
Can confirm it just crashes within a few seconds of being in a match. Here's a Proton Log file for Proton Experimental: 
[steam-1928420.log](https://github.com/user-attachments/files/21672166/steam-1928420.log)

**Comment by SteavenGamerYT** [comment]
same issue

**Comment by Yasumora** [comment]
I can confirm both Matchmaking and Ranked matches crash on loading screen or when the match starts. The other gamemodes (jetpack, deathmatch, etc) and training grounds seem to work fine.
Tested on CachyOS with proton cachyos native, tho other proton versions seem to be having the same issue, will send a more detailed report later.

**Comment by VelorumS** [comment]
Proton EM-10.0-2C has the fixes.

**Comment by nicklozon** [comment]
> Proton EM-10.0-2C has the fixes.

~~It does not.~~ Proton EM is a fork, my mistake. Will give it a try.

**Comment by nicklozon** [comment]
With some help from a friend, he found that the Unreal Engine minidump file failed with an access violation in the Agora RTC SDK.

```
Crash reason:  EXCEPTION_ACCESS_VIOLATION_READ
Crash address: 0x0000000000000028
Process uptime: not available

Thread 266  (crashed) - tid: 2008
 0  agora_rtc_sdk.dll + 0xfffd88
     rax = 0x00004043e24de5d9    rdx = 0x0000000000000000
     rcx = 0x0000000000000028    rbx = 0x000000009a41af70
     rsi = 0x0000000000000000    rdi = 0x0000000000000028
     rbp = 0x00000000911cd560    rsp = 0x00000000911cd400
      r8 = 0x000000009d2e21b8     r9 = 0x00000000000007d0
     r10 = 0x0000000000002e2e    r11 = 0x8101010101010100
     r12 = 0x00000000911cd6f0    r13 = 0x0000000000000000
     r14 = 0x000000009d2e21b8    r15 = 0x000000009d2e21b8
     rip = 0x00006ffff9dbfd88
```

This could be related to voice or game capture - I do see Farlight has a "replay" system now.

**Comment by mimorational** [comment]
_if any1 interested_ 
Clearing shader cache seems to increase performance alot, but micro freezes remain (not related to proton afaik)

> The other gamemodes (jetpack, deathmatch, etc) and training grounds seem to work fine.

Yeah, sticking to DM until devs fix their undertested game

**Comment by BananaWorks07** [comment]
> > Proton EM-10.0-2C has the fixes.
> 
> It does not.

Huh? It works fine for me (and at least one other person who asked to try)

**Comment by BananaWorks07** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/6717#issuecomment-3168182276

Yes agora tries to dereference RCX and as you can see it's value is 0x28. Adding a couple exports into iphlpapi works around the issue. It definetly is an application bug, but very simple to work around it

As for your micro freeze issue, I cannot reproduce it either.

**Comment by nicklozon** [comment]
> Huh? It works fine for me (and at least one other person who asked to try)

Maybe I'm misunderstanding what issue or Proton version you're referring to.

To clarify: 10.0-2c (beta) does not fix the access violation issue causing the crash for me. My proton log shows `Proton: 1753360060 proton-10.0-2c` and I'm still getting the access violation and the game crashes. Please note that the crash only occurs in the regular matchmaking game modes, with players present - I had a friend play his first ever match successfully on linux, then his second match crashed.

> Adding a couple exports into iphlpapi works around the issue. It definetly is an application bug, but very simple to work around it

So, are you using the workaround? Or is 10.0-2 working without the workaround?

**Comment by BananaWorks07** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/6717#issuecomment-3168735237

Oh I probably should not name my proton the same as valve's, wasn't aware that they also used 10.0-2C. Anyway, they are different versions you can find my 10.0-2C in my fork of Proton

**Comment by nicklozon** [comment]
> Oh I probably should not name my proton the same as valve's, wasn't aware that they also used 10.0-2C. Anyway, they are different versions you can find my 10.0-2C in my fork of Proton

Ah, I see now, the "EM" refers to your forked Proton repository. TY for clarifying, and for your work fixing it. Will give it a try.

**Comment by Yasumora** [comment]
I can confirm the fork "Proton EM-10.0-2C" has the fixes and battle royale works properly.

**Comment by mimorational** [comment]
I also confirm that unranked BR works with that version (yet to check ranked but must be the same)

Copy proton-EM-10.0-2C to /home/u s e r/.steam/steam/compatibilitytools.d/

**Comment by gofman** [comment]
This should work once again with Proton Experimental ([bleeding-edge] beta branch currently, see https://github.com/ValveSoftware/Proton/wiki/Proton-Versions#proton-bleeding-edge).

**Comment by pollux78** [comment]
Game freezes when playing with friends after around a hour, sometimes happens right away also when joining a match with friends

Cachyos, Mesa 25.2.1, Kernel 6.16.4, bleeding edge experimental

[steam-1928420.log](https://github.com/user-attachments/files/22057290/steam-1928420.log)

**Comment by kisak-valve** [comment]
Farlight 84 crashes after 1 match

Issue transferred from https://github.com/ValveSoftware/Proton/issues/9017.
@consciousY posted on 2025-08-31T02:21:06:

# Compatibility Report
- Name of the game with compatibility issues: Farlight 84
- Steam AppID of the game: 1928420

## System Information
- GPU: RX 5500xt 4gb
- Video driver version: Mesa 25.2.1
- Kernel version: 6.16.2
- Link to full system information report as [Gist](https://gist.github.com/): [Information](https://gist.github.com/consciousY/fe4a55e1cb807e574ea170dbf0b976d8)
- Proton version:  every proton version available there

## I confirm:
- [✅] that I haven't found an existing compatibility report for this game.
- [✅] that I have checked whether there are updates for my system available.

proton log : https://paste.cachyos.org/p/5b00556.log
cachyos bug report: https://paste.cachyos.org/p/7455715.log


## Symptoms 
so, I've been trying to play farlight 84 for  from the new global relaunch. every time i start the game i can play 1st match without any issue but then on the 2nd match i always get crashed.
I've tried multiple proton version like GE, EM, experimental including cachyos proton. all result the same issue. tried to get chatgpt but it didn't help at all.

## Reproduction

Play a match then start another match it gets crashed during the character selection phase or after the selection phase.

**Comment by GoranKovac** [comment]
[steam-1928420.log](https://github.com/user-attachments/files/22068137/steam-1928420.log)
Crashes randomly in matchmaking (with people only) no matter what proton used (EM,GE, experimental)...
System info
https://gist.github.com/GoranKovac/99f01ef961d63a1ffade336e84e28acb

**Comment by gofman** [comment]
A tentative fix for the latest issue with game crashing (or maybe hanging) randomly around joining a match is in Proton Experimental (bleeding-edge branch, see https://github.com/ValveSoftware/Proton/wiki/Proton-Versions#proton-bleeding-edge for the details how to select this Proton branch),

**Comment by GoranKovac** [comment]
> A tentative fix for the latest issue with game crashing (or maybe hanging) randomly around joining a match is in Proton Experimental (bleeding-edge branch, see https://github.com/ValveSoftware/Proton/wiki/Proton-Versions#proton-bleeding-edge for the details how to select this Proton branch),

Yes this one is working perfectly! Thank you

**Comment by Kaiamaterasu** [comment]
I use arch linux..
Farlight 84 
App id 1928420
Build id 19839027
Plasma latest
Ryzen 7 
Rx 6500 m
The game gets stuck after 2 matches... sometimes at 2nd match itself...
I have to restart everytime
..
Tried proton em
Proton experimental bleeding edge
Proton hotfix
Proton ge
Same issue at all.

**Comment by simifor** [comment]
@Kaiamaterasu Switch to experimental bleeding edge, add this to your game's launch parameter: `PROTON_LOG=1 %command%` and play the game until the issue happens, then upload the resulting `steam-1928420.log` file that will appear in your home folder.

**Comment by pollux78** [comment]
Hi, somehow me and my friend have been banned from the game, i made sure my team was fully wiped out and left the game, just a warning to anyone else who is gonna play this game on linux, if this is something to do with proton or not i am not sure

<img width="607" height="497" alt="Image" src="https://github.com/user-attachments/assets/65485566-203f-4370-aab0-4c88d8dc92c4" />

**Comment by nicklozon** [comment]
Proton GE and Proton EM both worked, but I did get the occasional freeze/crash which may be system related as it happens in Windows as well.

Experimental Bleeding Edge works perfect - played "first person classic ranked" for a couple hours on Tuesday. Will turn on debugging and continue playing some more, but for now, the initial issues are resolved for me.

**Comment by nicklozon** [comment]
> Hi, somehow me and my friend have been banned from the game, i made sure my team was fully wiped out and left the game, just a warning to anyone else who is gonna play this game on linux, if this is something to do with proton or not i am not sure

@pollux78 you'll have to contact the Farlight devs - they have a discord where you can open an issue I believe.

**Comment by GoranKovac** [comment]
@pollux78 I really hope its not Linux bans.... please keep us posted. Been playing for 5 days without any problems but now afraid of ban....

**Comment by mayronH** [comment]
> [@pollux78](https://github.com/pollux78) I really hope its not Linux bans.... please keep us posted. Been playing for 5 days without any problems but now afraid of ban....

3 people reporting 24hr ban today on their discord. The new update probably changed something. Got banned using Proton EM latest version.

**Comment by GoranKovac** [comment]
Well here goes nothing, game night with friends will see what happens (proton experimental bleeding edge)

**Comment by GoranKovac** [comment]
And got ban but I did not even played... just stayed in menu......

**Comment by VelorumS** [comment]
Yea, insta "account banned", "mode unavailable" and 24h countdown.

**Comment by GoranKovac** [comment]
Got a general respond from devs on discord that its "not supported" on linux so its better not to play it via linux because there were a lot of ban reports. Shame...another BR not playable

**Comment by Kaiamaterasu** [comment]
> Got a general respond from devs on discord that its "not supported" on linux so its better not to play it via linux because there were a lot of ban reports. Shame...another BR not playable

Wait...so we shouldn't play farlight on linux ???

**Comment by GoranKovac** [comment]
I don't know what to tell you... I played fine for 7 days and after latest patch I was in menu waiting for my friend and after few minutes account ban appeared so I was not even playing.

Respond (general) from dev in their discord:
```
Farlight 84 is only supported on PC (Windows). There is no official Linux version at this time. It is recommended that you use the officially supported version to avoid affecting the use of your account.
```

And on that discord many others reported ban also (on Linux).

**Comment by Kaiamaterasu** [comment]
> I don't know what to tell you... I played fine for 7 days and after latest patch I was in menu waiting for my friend and after few minutes account ban appeared so I was not even playing.
> 
> Respond (general) from dev in their discord:
> ```
> Farlight 84 is only supported on PC (Windows). There is no official Linux version at this time. It is recommended that you use the officially supported version to avoid affecting the use of your account.
> ```

But I have been playing since the day it was launched..I didn't get any ban....I will stop opening the game for now 
Thanks for letting me know..
And i don't know why these gaming industry hates linux so much..

**Comment by GoranKovac** [comment]
Did you played after yesterdays patch? BTW Its not permanent ban its only for 24h but I don't want to risk anymore.

**Comment by Kaiamaterasu** [comment]
> Did you played after yesterdays patch? BTW Its not permanent ban its only for 24h but I don't want to risk 

Yeah I played yesterday whole day.

And even this morning till afternoon..
I didn't get any ban

**Comment by GoranKovac** [comment]
What proton are you using to play it? I'm on proton experimental bleeding edge but other users that got it use various others like proton-cachyos, GE, EM etc

**Comment by Kaiamaterasu** [comment]
> What proton are you using to play it? I'm on proton experimental bleeding edge

Same...I use proton experimental bleeding edge

**Comment by pollux78** [comment]
> Got a general respond from devs on discord that its "not supported" on linux so its better not to play it via linux because there were a lot of ban reports. Shame...another BR not playable

Support doesn't know what they are talking about, if you open a new support ticket asking if it is supported with steamdeck they would reply with yes we are steamdeck verified. which runs under proton which means they support Linux unofficially aswell, it is the same response you get with other games, yes the game isn't natively supported on Linux but they have the anticheat enabled for proton so that it can work with Linux devices like the steamdeck.

**Comment by Kaiamaterasu** [comment]
> > Got a general respond from devs on discord that its "not supported" on linux so its better not to play it via linux because there were a lot of ban reports. Shame...another BR not playable
> 
> Support doesn't know what they are talking about, if you open a new support ticket asking if it is supported with steamdeck they would reply with yes we are steamdeck verified. which runs under proton which means they support Linux unofficially aswell, it is the same response you get with other games, yes the game isn't natively supported on Linux but they have the anticheat enabled for proton so that it can work with Linux devices like the steamdeck.

So the answer is yes and no both at same time.

**Comment by GoranKovac** [comment]
@pollux78 I am all aware of that, but what about bans? I've even sent a email and we just go in circles "we dont support linux". I can't get into discussion with them what they support and what they don't and explaining what proton is and what native port is. They could respond we will look into it regarding proton maybe false positive bans whatever that was also happening with apex but they just respond `we dont support linux` . I dont know the way forward with this but will certainly not risk getting a long or a permanent ban to test "maybe it works now"

**Comment by pollux78** [comment]
> [@pollux78](https://github.com/pollux78) I am all aware of that, but what about bans? I've even sent a email and we just go in circles "we dont support linux". I can't get into discussion with them what they support and what they don't and explaining what proton is and what native port is. They could respond we will look into it regarding proton maybe false positive bans whatever that was also happening with apex but they just respond `we dont support linux` . I dont know the way forward with this but will certainly not risk getting a long or a permanent ban to test "maybe it works now"

it could simply be a bug with there new update, it is better to let valve's proton team handle this situation, the support team should be fully aware that the game supports the steamdeck as it is verified, lots of game studio support teams do not relies that the steamdeck is linux and is using a translation layer to play the game, and there is nothing stopping a user from playing it on linux desktop itself unless the studio enforces some checks for steamdeck only which hasn't been said by any developer from this studio.

I myself have been false banned on apex and the EA support also were unaware of linux users playing the game but respawn did know and were working with valve to solve issues with proton and i got unbanned without contacting respawn as i wasnt able to anyways, same goes with embark studios with the finals, its all the same with every support team, all you can do like you said not play until there is something officially said about it either from the game studio or valve in here.

**Comment by GoranKovac** [comment]
Just to add an update there seems to be ban frenzy going on for linux, more and more people are reporting so everyone be careful

**Comment by Kaiamaterasu** [comment]
> Just to add an update there seems to be ban frenzy going on for linux, more and more people are reporting so everyone be careful

Buddy...I just got banned this evening..
For 24 hours

**Comment by ghost** [comment]
Anyone got an update on this? I also got affected by this. Using ProtonGE latest and got handed my first account ban of 24 hours over the game crashing 3 times.

**Comment by GoranKovac** [comment]
Does not matter which proton you use... the ban hammer will get you (they are banning linux folks).
They expanded the unsupportness: Linux, VM, compatibility layers etc (by their CS on discord)

EDIT: Seems there some progress but not 100% sure

**Comment by pollux78** [comment]
> Does not matter which proton you use... the ban hammer will get you (they are banning linux folks).
> They expanded the unsupportness: Linux, VM, compatibility layers etc (by their CS on discord)

Based on the other discord text channels many users can play again, my friend has been playing it all day and hasn't been banned 

https://discord.com/channels/795501514730635305/1413224155498676366

**Comment by Kaiamaterasu** [comment]
> > Does not matter which proton you use... the ban hammer will get you (they are banning linux folks).
> > They expanded the unsupportness: Linux, VM, compatibility layers etc (by their CS on discord)
> 
> Based on the other discord text channels many users can play again, my friend has been playing it all day and hasn't been banned 
> 
> https://discord.com/channels/795501514730635305/1413224155498676366
> 
> 

Is it confirmed?

**Comment by GoranKovac** [comment]
Yea it seems, was not aware of that since I've left discord but rejoined

**Comment by Kaiamaterasu** [comment]
> Yea it seems, was not aware of that since I've left discord but rejoined

...
I will be waiting for someone to play and say here..
That it works

**Comment by GoranKovac** [comment]
I've played one now but firstly stayed in the menu about 20min... so far works ¯\_(ツ)_/¯

**Comment by pollux78** [comment]
> > Yea it seems, was not aware of that since I've left discord but rejoined
> 
> ...
> I will be waiting for someone to play and say here..
> That it works 

I have played some matches with that friend now in ranked and have not been banned, as another user said I think it's ok for now, the support team continues to deny supporting Linux or even the steamdeck but someone is allowing it in the studio it seems, gotta love support who are uninformed 👍

**Comment by GoranKovac** [comment]
played 5-6 matches SEEMS fine™

**Comment by rezzafr33** [comment]
I got a 24-hour ban on Farlight84 after playing just one match on Bazzite, Proton Experimental

**Comment by VelorumS** [comment]
Crashes sometimes after a fight. It started after the 2025-09-11 update.

[steam-1928420.zip](https://github.com/user-attachments/files/22357193/steam-1928420.zip)

**Comment by ghost** [comment]
After one match using the newly updated Proton Experimental version, no bans or hard crashes yet, but I will perform 2 more matches to confirm if the new changes did fix the Multiplayer.s

I made sure that Proton EasyAntiCheat Runtime was installed, however, I am unsure since the game uses the DRM Protection module from EAC and not the Anti Cheat module.

**Comment by ghost** [comment]
Also, no launch parameters were used during those matches.

**Comment by Kaiamaterasu** [comment]
> Also, no launch parameters were used during those matches.

So right now at present is it safe to play farlight at linux right now.
And based on your observation which proton is better.
.and is there any recommendations or suggestions u would love to give..
And thank you so much for this help as fellow linux guy

**Comment by ghost** [comment]
> > Also, no launch parameters were used during those matches.
> 
> So right now at present is it safe to play farlight at linux right now. And based on your observation which proton is better. .and is there any recommendations or suggestions u would love to give.. And thank you so much for this help as fellow linux guy

From my point of view, yes, it seems safe to play.

I am using Linux Mint 22.2 with XFCE on Kernel 6.14 LTS. No further problems, except for a couple of hard crashes.

I used ProtonGE 15, no issues there, now I am doing 2 matches to confirm if the updated Proton Experimental fixed the multiplayer issue.

**Comment by ghost** [comment]
No suggestions from me sadly.

**Comment by Kaiamaterasu** [comment]
> No suggestions from me sadly.

It's okey...thank you so much for the information.

It was very valuable

**Comment by ghost** [comment]
> > No suggestions from me sadly.
> 
> It's okey...thank you so much for the information.
> 
> It was very valuable

You are very welcome

**Comment by ghost** [comment]
After 4 consistent matches, I can confirm

On my side

That I did not get a 24 hour ban and the game did not hard crash. The newest update to Proton Experimental did fix the multiplayer.

As a closing note: For performance related issues, the devs mentioned that on October, they plan to drop a massive Performance Optimization.

**Comment by ghost** [comment]
Scratch that, I am now suffering from consistent crashes

**Comment by ghost** [comment]
Got the logfile containing the crash.

Archived due to the log file exceeding Github's maximum allowed file size.

[steam-1928420.log.tar.gz](https://github.com/user-attachments/files/22393309/steam-1928420.log.tar.gz)

**Comment by VelorumS** [comment]
I'd encourage the ones who can debug Wine issues to apply for the current Farlight beta test. The application link is in the in-game mailbox.

Maybe we'll catch more crashes before the actual release.

**Comment by ghost** [comment]
> I'd encourage the ones who can debug Wine issues to apply for the current Farlight beta test. The application link is in the in-game mailbox.
> 
> Maybe we'll catch more crashes before the actual release.

On it.

Registration got sent over, awaiting a response.

**Comment by ghost** [comment]
Update on the Beta Test, I sadly did not get accepted. Let me know how well it goes.

**Comment by ghost** [comment]
My closing comment: Because of the unfair ban handed to my account, I am automatically not allowed to participate in the Beta Tests, so I wish those who do get accepted goodluck and let us know how it goes.

**Comment by Yasumora** [comment]
> My closing comment: Because of the unfair ban handed to my account, I am automatically not allowed to participate in the Beta Tests, so I wish those who do get accepted goodluck and let us know how it goes.

I got access to the playtest version of farlight, idk if I should open another issue or if posting stuff for the playtest version can be sent here. 

Either way the game wont open on proton experimental or proton cachy-os. 

On **proton experimental** I got this message:

<img width="331" height="178" alt="Image" src="https://github.com/user-attachments/assets/d26d2134-1be0-4027-a9e4-53295adb510c" />

then the game opened and crashed before finishing the "Lilith Games" banner animation, and got this log

[steam-3060610-PROTON_EXPERIMENTAL.log](https://github.com/user-attachments/files/22699344/steam-3060610-PROTON_EXPERIMENTAL.log)

I validated all files and they didn't seem to have any issues, "all 16315 files successfully validated". After Validating I tried launching the game again and it crashed on the same "Lilith Games" banner.

on **Proton 9.0.4** the game opened and crashed without a message at the same place it crashed on Proton experimental.

[steam-3060610-PROTON9.0.4.log](https://github.com/user-attachments/files/22699377/steam-3060610-PROTON9.0.4.log)

on **Proton 10.0-2** the game opened and crashed without a message at the same place it crashed on Proton experimental and Proton 9.0.4

[steam-3060610-PROTON10.0-2.log](https://github.com/user-attachments/files/22699390/steam-3060610-PROTON10.0-2.log)

**Comment by kisak-valve** [comment]
Farlight 84 crashes at Lilith splash screen after latest game update (AppID 1928420)

Issue transferred from https://github.com/ValveSoftware/Proton/issues/9103.
@linuxgamerlife posted on 2025-10-16T13:42:29:

[steam-1928420.log](https://github.com/user-attachments/files/22950611/steam-1928420.log)

# Compatibility Report
- Name of the game with compatibility issues: Farlight 84
- Steam AppID of the game: 1928420

## System Information
- GPU: AMD Radeon RX 9060 XT (RADV GFX1200)
- Video driver version: Mesa 25.1.9 (RADV)
- Kernel version: 6.16.11-200.fc42.x86_64
- Link to full system information report as Gist: https://gist.github.com/linuxgamerlife/a82d769b893d55fa03f55d20fe30cb95
- Proton version: Proton 10 Beta, Proton Experimental, GE-Proton10-17, GE-Proton10-20

## I confirm:
✅ that I haven't found an existing compatibility report for this game.
✅ that I have checked whether there are updates for my system available.

## Symptoms
After the most recent Farlight 84 update, the game no longer launches under Proton.
It displays the Lilith Games splash screen, then immediately closes to the desktop.

The Proton log shows an EXCEPTION_ACCESS_VIOLATION inside llhpbase.dll (launcher/protection module).
There are no Vulkan or DXVK errors; GPU initialization completes successfully.
CrashReportClient runs immediately after the game exits.

This occurs across all tested Proton versions, including ones where the game previously worked (GE-Proton10-17, Proton Experimental).

## Reproduction
1. Launch Farlight 84 on Linux using any current Proton version.
2. Wait for the Lilith Games splash screen.
3. The game closes before reaching the login screen.

**Comment by Kaiamaterasu** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/6717#issuecomment-3411131089

Well buddy.. because I have exactly the same issue....right now...

I am also hoping for a solution..or atleast i wanna know what's the reason...are we not allowed to play this game anymore as linux users .

**Comment by linuxgamerlife** [comment]
> Well buddy.. because I have exactly the same issue....right now...
> 
> I am also hoping for a solution..or atleast i wanna know what's the reason...are we not allowed to play this game anymore as linux users .

They are open about the fact they do not support Linux in one of their videos, which makes me wonder how they even got SteamDeck verified 🤔

**Comment by Kaiamaterasu** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/6717#issuecomment-3411247916

I have the same question...they say linux isn't allowed...and they don't support steam deck..but behind the scenes they do ..
 But what did it go wrong this time..
Can we expect repair from their side.
Or do we have to figure our way out...

**Comment by ghost** [comment]
I got some bad news, the new Season of Farlight 84 now instantly crashes when you launch it.

I believe this to be the final nail in the coffin for Linux support.

**Comment by linuxgamerlife** [comment]
They actually call out Linux as being bad in this video https://www.youtube.com/watch?v=hdo8vCJU0gw

**Comment by ghost** [comment]
> The actually call out Linux as being bad in this video https://www.youtube.com/watch?v=hdo8vCJU0gw

This is.....................heartbreakingly not the news I wanted to hear.

I cant believe I actually called it. This is a complete nightmare.

**Comment by Kaiamaterasu** [comment]
> They actually call out Linux as being bad in this video https://www.youtube.com/watch?v=hdo8vCJU0gw

I guess that's it...we are done with farlight 84

**Comment by VelorumS** [comment]
Yes, the video. But did they actually manage to block Linux or it's just another crash of a new update?

**Comment by Kaiamaterasu** [comment]
> Yes, the video. But did they actually manage to block Linux or it's just another crash of a new update?

Nice question to be honest

**Comment by Evansch0** [comment]
when i ran proton log there was a few things that popped out to me feel free to pop in here if any of these are unimportant at all.

597.440:0164:0390:err:dmo:create_placeholder_file Creating tag file h264-used.
0:00:00.020915783 12839 0x7fe7a01b5200 ERROR aacparse gstaacparse.c:358:gst_aac_parse_sink_setcaps: Need codec_data for raw AAC
0:00:00.020939017 12839 0x7fe7a01b5200 ERROR aacparse gstaacparse.c:358:gst_aac_parse_sink_setcaps: Need codec_data for raw AAC
597.572:0164:02ac:err:sync:RtlLeaveCriticalSection section 000000000122D8E8 "?" is not acquired
597.572:0164:02ac:err:sync:RtlLeaveCriticalSection section 000000000122D8E8 "?" is not acquired
602.948:0110:0140:err:service:device_notify_proc failed to get event, error 1726

**Comment by Kaiamaterasu** [comment]
> when i ran proton log there was a few things that popped out to me feel free to pop in here if any of these are unimportant at all.
> 
> 597.440:0164:0390:err:dmo:create_placeholder_file Creating tag file h264-used.
> 0:00:00.020915783 12839 0x7fe7a01b5200 ERROR aacparse gstaacparse.c:358:gst_aac_parse_sink_setcaps: Need codec_data for raw AAC
> 0:00:00.020939017 12839 0x7fe7a01b5200 ERROR aacparse gstaacparse.c:358:gst_aac_parse_sink_setcaps: Need codec_data for raw AAC
> 597.572:0164:02ac:err:sync:RtlLeaveCriticalSection section 000000000122D8E8 "?" is not acquired
> 597.572:0164:02ac:err:sync:RtlLeaveCriticalSection section 000000000122D8E8 "?" is not acquired
> 602.948:0110:0140:err:service:device_notify_proc failed to get event, error 1726

These are just warning at most

**Comment by linuxgamerlife** [comment]
> Yes, the video. But did they actually manage to block Linux or it's just another crash of a new update?

I am hoping it's a codec issue or something as I believe the video they play after the splash screen has always been glitchy and I believe they have changed it. They also mention something about animation issues in their patch notes.

>Due to an issue affecting the intro animations of certain hero skins (Mission Express, Ultimate Weapon, Starry Wander, Venus, Vanguard Commander), we have temporarily disabled the related animation display feature. The issue is currently being fixed, and the animations will be re-enabled once the fix is complete.

🤞🏼

**Comment by linuxgamerlife** [comment]
Ok I can confirm that it works without forcing any proton version (and if you do) if you use SteamDeck=1 %command% in launch options.

**Comment by Evansch0** [comment]
> Ok I can confirm that it works without forcing any proton version (and if you do) if you use SteamDeck=1 %command% in launch options.

ok why does this just work.

**Comment by linuxgamerlife** [comment]
> > Ok I can confirm that it works without forcing any proton version (and if you do) if you use SteamDeck=1 %command% in launch options.
> 
> ok why does this just work.

Maybe their anti-cheat is checking to see if you are playing on steamdeck, and this launch option fools it?

**Comment by ghost** [comment]
> Ok I can confirm that it works without forcing any proton version (and if you do) if you use SteamDeck=1 %command% in launch options.

Will try on my end and report back when I can. These install sizes drives me nuts XD

**Comment by ghost** [comment]
> Ok I can confirm that it works without forcing any proton version (and if you do) if you use SteamDeck=1 %command% in launch options.

Yea, no dice on this one. Still instantly crashes and refuses to launch.

**Comment by PPFHD** [comment]
Running without problem with proton-EM-10.0-30 and Launch options SteamDeck=1

**Comment by PPFHD** [comment]
> > Ok I can confirm that it works without forcing any proton version (and if you do) if you use SteamDeck=1 %command% in launch options.
> 
> Yea, no dice on this one. Still instantly crashes and refuses to launch.

Running without problem with proton-EM-10.0-30 and Launch options SteamDeck=1

**Comment by ghost** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/6717#issuecomment-3418421226

Oh? I was double checking if I miss-typed the launch flag. I always thought it is `STEAMDECK=1 %command%`

**Comment by Evansch0** [comment]
> > Replying to [#6717 (comment)](https://github.com/ValveSoftware/Proton/issues/6717#issuecomment-3418421226)
> 
> Oh? I was double checking if I miss-typed the launch flag. I always thought it is `STEAMDECK=1 %command%`

thats what im using proton EM 10.0-30

**Comment by ghost** [comment]
Using Proton Experiment and saw a medium sized black box top of the game.

I am also aware that Proton Experimental has this recurring video issue but I dont find it annoying enough and it does not break the game that badly.

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/80dfaade-38d3-4f04-88d1-b433b302e6e3" />

**Comment by ghost** [comment]
Would this be a Proton issue or a game issue as that is not my GPU?

<img width="810" height="194" alt="Image" src="https://github.com/user-attachments/assets/cd597b51-a80b-40f8-a2e8-614891634b22" />

**Comment by PPFHD** [comment]
> Using Proton Experiment and saw a medium sized black box top of the game.
> 
> I am also aware that Proton Experimental has this recurring video issue but I dont find it annoying enough and it does not break the game that badly.
> 
> <img alt="Image" width="1920" height="1080" src="https://private-user-images.githubusercontent.com/201666263/507082394-80dfaade-38d3-4f04-88d1-b433b302e6e3.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjE3NTc3MjMsIm5iZiI6MTc2MTc1NzQyMywicGF0aCI6Ii8yMDE2NjYyNjMvNTA3MDgyMzk0LTgwZGZhYWRlLTM4ZDMtNGYwNC04OGQxLWI0MzNiMzAyZTZlMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMDI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTAyOVQxNzAzNDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iMjk1NjQ5OTY0ZTA4ZmRhNWYzYTZiZWFhNTNhNmQxYjdmYTQ0MmJiMGZhNTBiNzMyMWQ3ODVkYmI2OWFhNDM0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.y2gl_PFXHq3al-eMrOUpzhDQrjYk8wP1kkiAopLgjBI">

With Proton EM-10.0-30 there is video on startup. The gpu reported as INVALID because Its Proton or Steam deck compatible mode. But uses your gpu performance.

**Comment by PPFHD** [comment]
> Would this be a Proton issue or a game issue as that is not my GPU?
> 
> <img alt="Image" width="810" height="194" src="https://private-user-images.githubusercontent.com/201666263/507157103-cd597b51-a80b-40f8-a2e8-614891634b22.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjE3NTc3MjMsIm5iZiI6MTc2MTc1NzQyMywicGF0aCI6Ii8yMDE2NjYyNjMvNTA3MTU3MTAzLWNkNTk3YjUxLWE4MGItNDBmOC1hMmU4LTYxNDg5MTYzNGIyMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMDI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTAyOVQxNzAzNDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00OTM2NjNjMmUwMDI3ZDE4MjEwM2I2NTBhYjViZDFkZmU5ODY5YzUyNDVmZjQwYmM1MjAwYmI2MGI0ZDk4NmI0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.wou9Tk_CioLlFaPUJk2gctfDHCNrbtqiYt9WHoKNyHs">

I installed Proton EM-10.0-30 with ProtonPlus application.

**Comment by simifor** [comment]
@MyNamesTee the video is working on my end, do you have shader precaching enabled? As for the black strip, if it only appears for a few seconds it must be what appears alongside the signin notification, it's not specific to your setup

**Comment by ghost** [comment]
> [@MyNamesTee](https://github.com/MyNamesTee) the video is working on my end, do you have shader precaching enabled? As for the black strip, if it only appears for a few seconds it must be what appears alongside the signin notification, it's not specific to your setup

I do not have Shader Pre-caching enabled as I never needed it.

**Comment by ghost** [comment]
I dont think this game is a Vulkan/OpenGL game

**Comment by simifor** [comment]
@MyNamesTee shader precaching isn't specific to those apis, and it's not specific to actual game shaders. Proton transcodes videos it can't play otherwise, this is delivered through the same mechanism.

**Comment by ghost** [comment]
> [@MyNamesTee](https://github.com/MyNamesTee) shader precaching isn't specific to those apis, and it's not specific to actual game shaders. Proton transcodes videos it can't play otherwise, this is delivered through the same mechanism.

Ahhh I see

**Comment by ghost** [comment]
So, I installed the non-Flatpak version of Steam and now Farlight shows...........this?

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/3b829f75-ba4b-4194-958e-76c4bfe371fc" />

**Comment by ghost** [comment]
I would like to apologize again........this scared me a little as I thought it was broken 

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/8abdcc4b-341e-4899-bf36-e284df5e8b1e" />

**Comment by ghost** [comment]
It seems the game is handing out these warnings as its new for me. Switched to Proton Experimental and still got this error. I have performed a File Integrity check but no dice on that.

<img width="1282" height="749" alt="Image" src="https://github.com/user-attachments/assets/6c272e70-c880-45fc-8d26-2381a754740a" />

**Comment by develOseven** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/6717#issuecomment-3168585829

> Yes agora tries to dereference RCX and as you can see it's value is 0x28. Adding a couple exports into iphlpapi works around the issue. It definetly is an application bug, but very simple to work around it

@Etaash-mathamsetty could you please explain more in detail where to add the workaround? A custom Wine build needs to be made?

**Comment by simifor** [comment]
@develOseven as far as I'm aware, the game should be working without extra steps, at the very least on proton experimental. Are there issues on your end?

**Comment by develOseven** [comment]
For me it doesn't work on Proton Experimental, but works on Proton-EM-10.0-32.

I think that 2025-11-21 Proton Experimental update broke something.

**Comment by simifor** [comment]
@develOseven just to be clear, by "doesn't work" do you mean that it crashes at random? I've been taking a look and I can play for random amounts of time but at some point it just crashes, I could see this on both experimental and proton 10. I compiled the previous experimental release and I could complete 4 matches on that one, so that's likely not affected.

**Comment by PPFHD** [comment]
Hi all!
I could run this game via PROTON-EM-10.0-37, It can connect and playable.
But one big problem: There is a Blank area in the upper 20% section, which created while the game notify me about Steam Connections in the login screen. I tried with SteamDeck and without SteamDeck options, but no chance.

**Comment by simifor** [comment]
@PPFHD on my desktop I see that the upper area becomes black under the notification, but it clears up after a few seconds, before the main menu of the game is done loading. Is this not the case for you?

**Comment by PPFHD** [comment]
> [@PPFHD](https://github.com/PPFHD) on my desktop I see that the upper area becomes black under the notification, but it clears up after a few seconds, before the main menu of the game is done loading. Is this not the case for you?

Mine isnt clear, Its an graphical bug, because i alt+tab-bing with different windows, Its shows the full screen about the game.. I use fedora kde plasma with wayland. Gpu is amd.

**Comment by simifor** [comment]
@PPFHD sorry but I have trouble understanding what you're saying, i t'd be preferable to share a video alongside more information about your setup like amd gpu model, mesa version, and using proton experimental.

For reference this is what I see (cropped to take less space here).
The game starts, shortly after that a login notification appears alongside a black background 

<img width="2560" height="881" alt="Image" src="https://github.com/user-attachments/assets/5552779b-77f2-4401-be32-53e83cbf36b8" />

After a few seconds the notification disappears as does the black background

<img width="2560" height="881" alt="Image" src="https://github.com/user-attachments/assets/7af7fd57-8fe9-46eb-a19b-c4dd8877bc42" />

I'm using proton experimental, rx 9070, mesa 26.1.0

**Comment by PPFHD** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/6717#issuecomment-4425389804

Mine is looks like too, than yours. But in menu and under game is black area visible too.

**Comment by simifor** [comment]
@PPFHD can you give more information about your card, mesa version and whether the issue happens with proton experimental?

**Comment by PPFHD** [comment]
> [@PPFHD](https://github.com/PPFHD) can you give more information about your card, mesa version and whether the issue happens with proton experimental?

Rx 5600 xt
mesa-dri-drivers-26.0.6-2.fc44
Fedora 44 kde plasma (wayland)
The Proton version is EM-10.0-37, not Experimental. The upper 20% of screen not visible in game.

**Comment by simifor** [comment]
Yes, but this is the proton issue tracker, proton em is a third party
project. If the problem only happens with proton em you need to report the
issue to them, if the issue also happens on experimental then comment again
stating that

**Comment by PPFHD** [comment]
> Yes, but this is the proton issue tracker, proton em is a third party
> project. If the problem only happens with proton em you need to report the
> issue to them, if the issue also happens on experimental then comment again
> stating that
> […](#)

I think with Experimental no video codec, or Experimental cant play the game startup videos, the screen like an old color tv no signal picture, so i cant login. No start button.

**Comment by simifor** [comment]
@PPFHD videos work on experimental, maybe you have shader cache disabled? steam settings > download
That said, After first connecting my account to the game it's never asked me to login manually

**Comment by PPFHD** [comment]
> [@PPFHD](https://github.com/PPFHD) on my desktop I see that the upper area becomes black under the notification, but it clears up after a few seconds, before the main menu of the game is done loading. Is this not the case for you?

Which Linux you use?

**Comment by simifor** [comment]
@PPFHD arch. And please confirm whether the game works on proton experimental or not and whether you have shader cache enabled

**Comment by linuxgamerlife** [comment]
Farlight 84 works fine with Proton 10.1000-105

Steam launch command:
```SteamDeck=1 %command%```

```

╭───────────────────── System ───────────────────────╮
│       Fedora Linux 43
│       7.0.5-cachyos1.fc43.x86_64
│       zsh 5.9
╰────────────────────────────────────────────────────╯
╭──────────────────── Hardware ──────────────────────╮
│       AMD Ryzen 7 5700X3D 8-Core Processor
│       Radeon RX 9070XT 16GB
│       8 GB / 31 GB
╰────────────────────────────────────────────────────╯
╭──────────────────── Session ───────────────────────╮
│       WAYLAND
│       niri
│       noctalia
│       kitty
╰────────────────────────────────────────────────────╯

**Comment by PPFHD** [comment]
> [@PPFHD](https://github.com/PPFHD) arch. And please confirm whether the game works on proton experimental or not and whether you have shader cache enabled

Shader cache enabled all time,
game integration checked,
 no running on experimental (click on play, and after some sec its automaticly stopped)

**Comment by simifor** [comment]
@PPFHD turns out this game uses h264 for videos, it's likely you don't have support for that enabled, you can get that sorted by running this in your terminal: `steam steam://unlockh264/`.

As for the crash itself, as you'd been swapping proton versions around, it's best if you wipe the game's prefix. Assuming it's in the default disk, it will be in `/home/youruser/steam/steam/steamapps/compatdata/1928420`, just delete that folder.

If crashing persists after that, change the game's launch parameter on steam to `PROTON_LOG=1 %command%` then attempt to run the game again, this will generate a text file called `steam-1928420.log` in your home folder, upload it here, and zip it if it happens to be too big for github.

**Comment by PPFHD** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/6717#issuecomment-4468009251

I tried all that 3 things, but nothing changed.

Log is only contain this:

```
======================
Proton: 1778138606 experimental-11.0-20260506b
SteamGameId: 1928420
Command: ['/run/media/ppfhd/noname/SteamLibrary/steamapps/common/Farlight 84/start_protected_launcher.exe']
Options: {'forcelgadd', 'gamedrive'}
depot: 4.0.20260415.225012
pressure-vessel: 0.20260414.0
scripts: 0.20260414.0
steamrt4: 4.0.20260415.225012 steamrt4 4.0.20260415.225012
Kernel: Linux 7.0.8-200.fc44.x86_64 #1 SMP PREEMPT_DYNAMIC Fri May 15 14:03:46 UTC 2026 x86_64
Language: LC_ALL None, LC_MESSAGES None, LC_CTYPE None
PATH: /home/ppfhd/.local/share/Steam/steamapps/common/Proton - Experimental/files/bin/:/usr/bin:/bin
Effective WINEDEBUG: +timestamp,+pid,+tid,+seh,+unwind,+threadname,+debugstr,+loaddll,+mscoree
======================
```

Game runs with Ge proton with PROTOn enable wayland launch option helps the 30% blank screen removing while logging in. So no problem with ge proton. but no start with experimental.

**Comment by simifor** [comment]
@PPFHD that's just the header for the proton log, if it ends there it means the didn't even get to run thus there's no log. Given that it's generally working for people, I'd first try moving the game to your main drive to avoid any potential filesystem shenanigans (specially if the partition it's in right now happens to be ntfs) and see if it that makes a difference. 

When there's no proton log populated there's also a chance you might get some explanation why in `~/.steam/steam/logs/console-linux.txt`.

Also, given that the proton log is empty that should mean that the behavior isn't what you described before, with the game starting but getting a color pattern, if it was getting that far currently then the logs would show something
